doc/METADATA.ini

```diff
@@ -905,3 +905,13 @@ default = dec
 usedby/plugins = hexnumber
 description = used to specify the base of an integer value. Currently only the hexnumber plugin supports this metadata. It converts any value marked as "hex" from
     hexadecimal into decimal when the configuration is read, and back when it is written. The hexnumber plugin also ensures that a value marked as "hex" starts with "0x" or "0X".
+
+[elektra/spec/regex/check/enum]
+type = string
+usedby/plugins = regexdispatcher
+description = the generated regex representing the type of check/enum/# and check/enum/multi metakeys
+
+[elektra/spec/regex/check/range]
```

Comment:
Some thoughts about the metadata name:

- the elektra prefix does not give information, all meta data is for and from Elektra.
- it should also not start with spec (do not be confused with the namespace).

And wouldn't it be better that we only have a single metadata and the regex is always merged together? Do you need the information of where the regex comes from?

What was the idea of giving this name?

Reply:
you are right. as the regexdispatcher does all the preprocessing this essentially yields a check/validation keyword in the end all the time, so in that case i don't need further information where it came from i think. i will try that idea out.